devicon.json

```diff
@@ -7814,6 +7814,31 @@
         "color": "#173aee",
         "aliases": []
     },
+    {
+        "name": "scikitlearn",
+        "altnames": [
+            "scikit-learn",
+            "scikit",
+            "sklearn"
+        ],
+        "tags": [
+            "machine-learning",
+            "python"
+        ],
```

Comment:
```suggestion
        "tags": [
            "machine-learning",
            "python",
            "tool"
        ],
```

Reply:
Thank you @lunatic-fox, I will start doing the same and optimize them prior to submission! I thought that optimization was part of the GitHub Actions.

Reply:
No problem! And you're right to think that way, but we're still developing that optimization to be a proper GitHub action. 🙂